src/control/server/server_utils.go

```diff
@@ -160,6 +160,26 @@ func getFabricNetDevClass(cfg *config.Server, fis *hardware.FabricInterfaceSet)
 	return netDevClass, nil
 }
 
+// Detect if any engines share numa nodes and if that's the case, allocate only on the shared numa
+// node and notify user.
+func getEngineNUMANodes(log logging.Logger, engineCfgs []*engine.Config) []string {
+	nodeMap := make(map[string]bool)
```

Comment:
Minor nit, but I think it's more idiomatic in Go to use an empty struct as the value than the bool. I know we've gone back and forth on it a bit.

Reply:
I think the opinion is split on this, in various authoritative articles I have seen people opt for bool because rather than `if _, exists := map[key]; exists {` to check for existence you can simply use `if map[key] {`. I would opt for using bool unless we have a strict style guide policy against it.